### DISCUSSION TITLE
AverageRating Filter Not Populating in Pnp Search Filters

### DISCUSSION THREAD

**Discussion by huetvictoria** [microsoft-search/pnp-modern-search#2544] [Q&A] [answered]
I am trying to add the AverageRating Managed Property to my Filters in my PNP Search Functionality. I confirmed that it was added as a Property and I have added it in the Custom Filters web part. I have reindexed the site, the column has data in it. Help!
![image](https://user-images.githubusercontent.com/67760631/201949247-b30b8964-6b27-4171-a5d3-6a72dd471f45.png)
![image](https://user-images.githubusercontent.com/67760631/201949465-ada1ebb8-a19c-41b9-9a83-3a820718fde8.png)

**Comment by wobba** [comment, marked as answer]
You can only use properties which are refinable. See https://learn.microsoft.com/en-us/sharepoint/technical-reference/crawled-and-managed-properties-overview and look at the "Refinable" column. You can also access schema management and see it there.  https://learn.microsoft.com/en-us/sharepoint/manage-search-schema

**Reply by huetvictoria** [reply]
Thanks @wobba - have there been any solutions where someone has mapped AverageRating to a RefineableDecimal and had it display correctly. I have done this but the values are showing like this, which is not helpful: 
![image](https://user-images.githubusercontent.com/67760631/202278156-2f027922-7207-4242-be0d-ab1f0fdd5ceb.png)

**Reply by wobba** [reply]
Map it to RefinableString instead
